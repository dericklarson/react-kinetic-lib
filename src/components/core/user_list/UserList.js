import React from 'react';

const UserList = props => <div className="user-list">User List</div>;

export default UserList;
