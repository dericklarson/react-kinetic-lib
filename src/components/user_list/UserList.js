import React from 'react';

export const UserList = props => <div className="user-list">User List</div>;
