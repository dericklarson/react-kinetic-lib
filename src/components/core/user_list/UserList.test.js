import React from 'react';
import { shallow } from 'enzyme';

import UserList from './UserList';

describe('<UserList />', () => {
  it('should render correctly', () => {
    const component = shallow(<UserList />);

    expect(component).toMatchSnapshot();
  });
});
