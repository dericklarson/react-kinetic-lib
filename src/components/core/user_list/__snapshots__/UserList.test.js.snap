// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserList /> should render correctly 1`] = `
<div
  className="user-list"
>
  User List
</div>
`;
